docs/content/preview/explore/query-1-performance/pg-stat-progress-create-index.md

```diff
@@ -0,0 +1,62 @@
+
+YugabyteDB supports the PostgreSQL `pg_stat_progress_create` view to report the progress of the CREATE INDEX command execution. Whenever [CREATE INDEX](../../../api/ysql/the-sql-language/statements/ddl_create_index/) is running, the `pg_stat_progress_create_index` view contains one row for each client connection that is currently running a CREATE INDEX command.
+
+The `pg_stat_progress_create_index` view can provide the following details:
+
+- number of rows processed during an index backfill.
+- the current phase of the command with `initializing` or `backfilling` as the possible phases.
+- index progress report for all the different configurations of an index or index build such as Nonconcurrent indexes, GIN indexes, Partial indexes, and Include indexes.
+
+The following table describes the view columns:
+
+| Column | Type | Description |
+| :----- | :--- | :---------- |
+| pid | integer | Process ID of backend that is running the CREATE INDEX. |
+| datid | OID | Object ID of the database to which this backend is connected. |
+| datname | name | Name of the database to which this backend is connected. |
+| relid | OID | Object ID of indexed relation.|
+| index_relid | OID | Object ID of index. |
+| command | text | The command that is running CREATE INDEX CONCURRENTLY, or CREATE INDEX NONCONCURRENTLY. |
+| phase | text | The current phase of the command. The possible phases are _initializing_, or _backfilling_. |
+| tuples_total | bigint | Number of indexed table tuples already processed. |
+| tuples_done | bigint | Estimate of total number of tuples (in the indexed table). This value is retrieved from `pg_class.reltuples`. |
+| partitions_total | bigint | If the ongoing CREATE INDEX is for a partitioned table, this refers to the total number of partitions in the table. Set to 0 otherwise. |
+| partitions_done | bigint | If the ongoing CREATE INDEX is for a partitioned table, this refers to the number of partitions the index has been created for. Set to 0 otherwise. |
+
+Columns such as `lockers_total`, `lockers_done`, `current_locker_pid`, `blocks_total`, and `blocks_done` are not applicable to YugabyteDB and will have always have null values.
+
+## YugabyteDB-specific changes
+
+The `pg_stat_progress_create_index` view includes the following YugabyteDB-specific changes:
+
+- In YugabyteDB, the `pg_stat_progress_create_index` view is a local view; it only has entries for CREATE INDEX commands issued by local YSQL clients.
+
+- In PostgreSQL, `tuples_done` and `tuples_total` refer to the tuples of the _index_. However, in YugabyteDB, because it is currently easier to get indexed table counts (and there's no effortless way to retrieve the index counts), these fields refer to the tuples of the _indexed table_. This discrepancy is only evident for partial indexes, where the reported progress will be less than the actual progress. `tuples_total` is an estimate that is retrieved from `pg_class.reltuples`.
```

Comment:
I think some of these details can be left out from the doc - specifically, 

> However, in YugabyteDB, because it is currently easier to get indexed table counts (and there's no effortless way to retrieve the index counts), these fields refer to the tuples of the _indexed table_.

We can specify the difference between PG and YB and leave it at that.